website/source/api/acl/role-binding-rules.html.md

```diff
@@ -0,0 +1,371 @@
+---
+layout: api
+page_title: ACL Role Binding Rules - HTTP API
+sidebar_current: api-acl-role-binding-rules
+description: |-
+  The /acl/rolebindingrule endpoints manage Consul's ACL Role Binding Rules.
+---
+
+-> **1.5.0+:**  The APIs are available in Consul versions 1.5.0 and later. The documentation for the legacy ACL API is [here](/api/acl/legacy.html)
+
+# ACL Role Binding Rule HTTP API
+
+The `/acl/rolebindingrule` endpoints [create](#create-a-role-binding-rule),
```

Comment:
Is it too late to rename this endpoint? The `rolebindingrule` endpoints it a little long and hard to read.

Reply:
What name would you suggest?

Reply:
I think we should bikeshed this a little more in Slack maybe, I also brought this up. Agree that `rolebindingrule` might be the best name for it, but if we can get a shorter one it might be better. If we can't come up with anything else between us then fairynuff 🤷‍♂️

Reply:
Maybe we can discuss on the community call Friday? My initial thoughts are:
- `binding` since role is already an endpoint. Also this seems like the only binding that we will have with the ACLs (we should think into the future). 
- `bindingrules` or `bind-rules` or `bindrules` this is a little more specific, but becomes harder to use. 
- `rules`, but this is in danger of being overloaded.